Netro Sprinklers.indigoPlugin/Contents/Server Plugin/plugin.py

```
@@ -270,15 +263,68 @@ def _make_api_call(self, url, request_method="get", data=None):
+                                    f"consider increasing polling interval"
+                                )
+
+                            self.logger.warning(error_msg)
+                            self._fireTrigger("rateLimitExceeded")
+                            raise ThrottleDelayError(error_msg)
+                        elif error.get("code") == 1:
+                            # Invalid key error
+                            self.logger.error(
+                                f"invalid netro serial number: {error.get('message')}, "
+                                f"verify the serial number in your device configuration"
+                            )
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)